App crashes when minimizing while playing certain videos

**Describe the bug**
When playing some video files (example below) the app crashes when trying to enter picture in picture mode

**To Reproduce**
1. Play a video
2. Press the home button

**Expected behavior**
The app should "minimize" and picture in picture mode should begin

**Logs**
```06-14 13:11:09.844 22865 22865 E AndroidRuntime: FATAL EXCEPTION: main
06-14 13:11:09.844 22865 22865 E AndroidRuntime: Process: org.jellyfin.mobile, PID: 22865
06-14 13:11:09.844 22865 22865 E AndroidRuntime: java.lang.IllegalArgumentException: enterPictureInPictureMode: Aspect ratio is too extreme (must be between 0.418410 and 2.390000).
06-14 13:11:09.844 22865 22865 E AndroidRuntime:        at android.os.Parcel.createExceptionOrNull(Parcel.java:2377)
06-14 13:11:09.844 22865 22865 E AndroidRuntime:        at android.os.Parcel.createException(Parcel.java:2357)
06-14 13:11:09.844 22865 22865 E AndroidRuntime:        at android.os.Parcel.readException(Parcel.java:2340)
06-14 13:11:09.844 22865 22865 E AndroidRuntime:        at android.os.Parcel.readException(Parcel.java:2282)
06-14 13:11:09.844 22865 22865 E AndroidRuntime:        at android.app.IActivityTaskManager$Stub$Proxy.enterPictureInPictureMode(IActivityTaskManager.java:6523)
06-14 13:11:09.844 22865 22865 E AndroidRuntime:        at android.app.Activity.enterPictureInPictureMode(Activity.java:2850)
06-14 13:11:09.844 22865 22865 E AndroidRuntime:        at org.jellyfin.mobile.player.PlayerFragment.onUserLeaveHint(PlayerFragment.kt:6)
06-14 13:11:09.844 22865 22865 E AndroidRuntime:        at org.jellyfin.mobile.MainActivity.onUserLeaveHint(MainActivity.kt:3)
06-14 13:11:09.844 22865 22865 E AndroidRuntime:        at android.app.Activity.performUserLeaving(Activity.java:8189)
06-14 13:11:09.844 22865 22865 E AndroidRuntime:        at android.app.Instrumentation.callActivityOnUserLeaving(Instrumentation.java:1520)
06-14 13:11:09.844 22865 22865 E AndroidRuntime:        at android.app.ActivityThread.performUserLeavingActivity(ActivityThread.java:4653)
06-14 13:11:09.844 22865 22865 E AndroidRuntime:        at android.app.ActivityThread.handlePauseActivity(ActivityThread.java:4637)
06-14 13:11:09.844 22865 22865 E AndroidRuntime:        at android.app.servertransaction.PauseActivityItem.execute(PauseActivityItem.java:46)
06-14 13:11:09.844 22865 22865 E AndroidRuntime:        at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:176)
06-14 13:11:09.844 22865 22865 E AndroidRuntime:        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:97)
06-14 13:11:09.844 22865 22865 E AndroidRuntime:        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2066)
06-14 13:11:09.844 22865 22865 E AndroidRuntime:        at android.os.Handler.dispatchMessage(Handler.java:106)
06-14 13:11:09.844 22865 22865 E AndroidRuntime:        at android.os.Looper.loop(Looper.java:223)
06-14 13:11:09.844 22865 22865 E AndroidRuntime:        at android.app.ActivityThread.main(ActivityThread.java:7660)
06-14 13:11:09.844 22865 22865 E AndroidRuntime:        at java.lang.reflect.Method.invoke(Native Method)
06-14 13:11:09.844 22865 22865 E AndroidRuntime:        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
06-14 13:11:09.844 22865 22865 E AndroidRuntime:        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
06-14 13:11:09.844 22865 22865 E AndroidRuntime: Caused by: android.os.RemoteException: Remote stack trace:
06-14 13:11:09.844 22865 22865 E AndroidRuntime:        at com.android.server.wm.ActivityTaskManagerService.ensureValidPictureInPictureActivityParamsLocked(ActivityTaskManagerService.java:4206)
06-14 13:11:09.844 22865 22865 E AndroidRuntime:        at com.android.server.wm.ActivityTaskManagerService.enterPictureInPictureMode(ActivityTaskManagerService.java:4089)
06-14 13:11:09.844 22865 22865 E AndroidRuntime:        at android.app.IActivityTaskManager$Stub.onTransact(IActivityTaskManager.java:3123)
06-14 13:11:09.844 22865 22865 E AndroidRuntime:        at android.os.Binder.execTransactInternal(Binder.java:1154)
06-14 13:11:09.844 22865 22865 E AndroidRuntime:        at android.os.Binder.execTransact(Binder.java:1123)
```

**Screenshots**
N/A

**System (please complete the following information):**
- App version: 2.3.0-beta.3
- Android version: 11 (LineageOS 18.1)
- Device manufacturer and model: OnePlus 3T
- Jellyfin server version: 10.7.5
- The type of video player implementation used: native player (ExoPlayer)

**Additional context**
Happens only when using ExoPlayer
video info:
```
Container mkv
Size 2476 MB
Video
Title 1080p H264 SDR
Codec H264
AVC No
Profile Main
Level 40
Resolution 1920x800
Aspect ratio 2.40:1
Interlaced No
Framerate 23.976025
Bitrate 3230 kbps
Bit depth 8 bit
Video range SDR
Color space bt709
Color transfer bt709
Color primaries bt709
Pixel format yuv420p
Ref frames 1
NAL 0
```

--- a/app/src/main/java/org/jellyfin/mobile/player/PlayerFragment.kt
+++ b/app/src/main/java/org/jellyfin/mobile/player/PlayerFragment.kt
@@ -53,6 +53,8 @@ import org.jellyfin.mobile.utils.Constants.DEFAULT_CENTER_OVERLAY_TIMEOUT_MS
 import org.jellyfin.mobile.utils.Constants.DEFAULT_CONTROLS_TIMEOUT_MS
 import org.jellyfin.mobile.utils.Constants.DEFAULT_SEEK_TIME_MS
 import org.jellyfin.mobile.utils.Constants.GESTURE_EXCLUSION_AREA_TOP
+import org.jellyfin.mobile.utils.Constants.PIP_MAX_RATIONAL
+import org.jellyfin.mobile.utils.Constants.PIP_MIN_RATIONAL
 import org.jellyfin.mobile.utils.SmartOrientationListener
 import org.jellyfin.mobile.utils.dip
 import org.jellyfin.mobile.utils.disableFullscreen
@@ -528,7 +530,14 @@ class PlayerFragment : Fragment() {
     private fun Activity.enterPictureInPicture() {
         if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
             val params = PictureInPictureParams.Builder().apply {
-                setAspectRatio(currentVideoStream?.aspectRational)
+                val aspectRational = currentVideoStream?.aspectRational?.let { aspectRational ->
+                    when {
+                        aspectRational < PIP_MIN_RATIONAL -> PIP_MIN_RATIONAL
+                        aspectRational > PIP_MAX_RATIONAL -> PIP_MAX_RATIONAL
+                        else -> aspectRational
+                    }
+                }
+                setAspectRatio(aspectRational)
             }.build()
             enterPictureInPictureMode(params)
         } else {

--- a/app/src/main/java/org/jellyfin/mobile/utils/Constants.kt
+++ b/app/src/main/java/org/jellyfin/mobile/utils/Constants.kt
@@ -1,6 +1,7 @@
 package org.jellyfin.mobile.utils
 
 import android.media.session.PlaybackState
+import android.util.Rational
 import org.jellyfin.mobile.BuildConfig
 
 @Suppress("MagicNumber")
@@ -95,6 +96,8 @@ object Constants {
     const val SCREEN_BRIGHTNESS_MAX = 255
     const val ZOOM_SCALE_BASE = 1f
     const val ZOOM_SCALE_THRESHOLD = 0.01f
+    val PIP_MIN_RATIONAL = Rational(100, 239)
+    val PIP_MAX_RATIONAL = Rational(239, 100)
     const val SUPPORTED_VIDEO_PLAYER_PLAYBACK_ACTIONS: Long = PlaybackState.ACTION_PLAY_PAUSE or
         PlaybackState.ACTION_PLAY or
         PlaybackState.ACTION_PAUSE or

